apps/services/user-profile/src/app/user-profile/dto/updateUserProfileDto.ts

```diff
@@ -0,0 +1,24 @@
+import { IsString, IsOptional } from 'class-validator'
+import { ApiPropertyOptional } from '@nestjs/swagger'
+
+export class UpdateUserProfileDto {
+  @IsOptional()
+  @IsString()
+  @ApiPropertyOptional()
+  readonly mobilePhoneNumber?: string
+
+  @IsOptional()
+  @IsString()
+  @ApiPropertyOptional()
+  readonly locale?: string
```

Comment:
just thinking if this should be an enum? we are going to support icelandic and english, then polish later. Just wondering if we should enforce the enum immediately

Reply:
Yeah makes sense

Reply:
@arnifreyrs Any opinion where this Enum should reside. Should maybe be part of the localization Lib. Or just simply within the user-profile to begin with?

Reply:
great question, It probably should reside in the localization lib, but that might cause some problems. I am fine by having it in the user-profile for now